CONTRIBUTING.md

```diff
@@ -6,6 +6,25 @@ You found something you'd like to change? Great! Please submit a pull request an
 2. Push to the branch (`git push origin my-new-feature`)
```

Comment:
We should probably call out that people should work from their forks.

Reply:
The bulleted list of "Commit your changes", "Push to branch" etc feels unnecessary. We are talking about common `git` steps which I believe we can assume a contributor is aware of already.

Reply:
That's true we could leave that out.

Reply:
Updated